api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -740,7 +755,292 @@ public void shouldTestPurgeProgramAttributeType() throws Exception {
                 int totalAttributeTypes = pws.getAllProgramAttributeTypes().size();
 		pws.purgeProgramAttributeType(programAttributeType);
 		assertEquals((totalAttributeTypes - 1), pws.getAllProgramAttributeTypes().size());
-}
+	}
+	
+	
+	@Test
+	public void getPrograms_shouldTestGetPrograms() {
+		List<Program> malPrograms = pws.getPrograms("MAL");
+		List<Program> prPrograms = pws.getPrograms("PR");
+		assertEquals(malPrograms.size(), 1);
+		assertEquals(prPrograms.size(), 3);
+	}
+	
+	@Test
+	public void retireProgram_shouldSetRetiredStateToFalseAndSetAReason() {
+		executeDataSet("org/openmrs/api/include/ProgramWorkflowServiceTest-otherProgramWorkflows.xml");
```

Comment:
Instead of doing this here, I would create a constant with the others, something like `protected static final OTHER_PROGRAM_WORKFLOWS = "org/openmrs/api/include/ProgramWorkflowServiceTest-otherProgramWorkflows.xml";`

Then add to the `runBeforeEachTest()` method:

`executeDataset(OTHER_PROGRAM_WORKFLOWS)`.

Reply:
Moved this in the method you specified